web/Auth/ClaimsTransformer.cs

```diff
@@ -60,7 +61,7 @@ private async Task<UserDbModel> CreateUserInDatabase(string googleAccountId, Cla
             GoogleUserName = name,
             GoogleEmail = email,
             GoogleAccountId = googleAccountId,
-            IsAdmin = false
+            IsAdmin = false,

```

Comment:
Not that it matters but any reason you added this comma? 😄

Reply:
I added it when I was trying something else but forgot to remove it oops